scripts/node_control_tool.py

```diff
@@ -14,30 +19,41 @@ def compose_cmd(cmd):
 
 def call_upgrade_script(version):
     import subprocess
-    print('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
-    try:
-        if test_mode:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
-        else:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node', version]), shell=True)
-        if retcode != 0:
-            print('Upgrade failed')
-    except:
-        print('Something went wrong with calling upgrade script')
+    logger.info('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
+    if test_mode:
+        retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
+    else:
+        retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node', version]), shell=True)
+    if retcode != 0:
+        msg = 'Upgrade failed: upgrade script returned {}'.format(retcode)
```

Comment:
Upgrade script is a script which performs the installation of new code. It's always the same. Migration scripts are optional, they can be applied after upgrade if some major changes appeared in the code, backwards compatibility was broken and system needs to update its state to work with the new code